src/test/java/com/faforever/client/game/GameBuilder.java

```diff
@@ -34,6 +35,7 @@ public GameBuilder defaultValues() {
     game.setMaxRating(800);
     game.setMaxRating(1300);
```

Comment:
This dupe a typo? The 800 will do nothing.  That a 'min' rating perhaps?

Reply:
This code is only used for testing. But yes this is probably a typo

Reply:
Make a PR if you like XD @dev-nulled heard u get t-shirts for those

Reply:
Sadly this one was opened too early so it doesn’t count :P